src/components/PeopleAndRoles/OrgPerson.vue

```diff
@@ -434,7 +476,7 @@ export default class OrgPerson extends Mixins(CommonMixin) {
   }
 
   private isRoleLocked (role: Roles): boolean {
-    return (this.orgPerson?.roles.some(party => party.roleType === role) && isNaN(this.activeIndex))
+    return (this.orgPerson?.roles.some(r => r.roleType === role) && !isNaN(this.activeIndex))
```

Comment:
I'm responsible for this bug from when I changed activeIndex from -1 to NaN :P

Reply:
I think the recommended approach to `isNaN` is to use `Number.sNaN()` due to some weird false positives you can get without. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN

Reply:
The problem is that I forgot to invert the condition when I changed the original code from `this activeIndex !== -1` to `isNaN(this.activeIndex)`.

This code now works fine.

Reply:
Sounds good!